Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
-    erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
       base64
       faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-http-cache (2.5.0)
+    faraday-http-cache (2.5.1)
       faraday (>= 0.8)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     faraday-net_http_persistent (2.1.0)
       faraday (~> 2.5)
       net-http-persistent (~> 4.0)
-    ffi (1.15.5)
+    ffi (1.16.3)
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    flay (2.12.0)
-      erubis (~> 2.7.0)
+    flay (2.13.2)
+      erubi (~> 1.10)
       path_expander (~> 1.0)
       ruby_parser (~> 3.0)
       sexp_processor (~> 4.0)
     flowdock (0.7.1)
       httparty (~> 0.7)
       multi_json
-    forking_test_runner (1.2.0)
+    forking_test_runner (1.13.0)
       parallel_tests (>= 1.3.7)
-    git_diff_parser (3.1.0)
-    gitlab (4.10.0)
-      httparty (~> 0.14, >= 0.14.0)
-      terminal-table (~> 1.5, >= 1.5.1)
-    globalid (1.1.0)
-      activesupport (>= 5.0)
+    git_diff_parser (3.2.0)
```

Comment:
### [git_diff_parser](https://github.com/packsaddle/ruby-git_diff_parser)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.1.0 → 3.2.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/git_diff_parser/3.1.0/3.2.0)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/packsaddle/ruby-git_diff_parser/compare/v3.1.0...v3.2.0).

These are the individual commits:
- (d679ad1) [Fix trailing whitespace issue in filename parsing](https://github.com/packsaddle/ruby-git_diff_parser/commit/d679ad1dc6e9f26792e8a2ea60aec328f39cdaf8)
- (69d3c5a) [Merge pull request #209 from daniel-beard/dbeard_fix_trailin…](https://github.com/packsaddle/ruby-git_diff_parser/commit/69d3c5a03cc84469533166480924db7f3f8ea03e)
- (ac2eaf3) [chore(test): test against ruby 2.5.3, 2.6.1](https://github.com/packsaddle/ruby-git_diff_parser/commit/ac2eaf3e59755ef6167691098d7689e716ce9abb)
- (89c1293) [chore(deps): byebug requires ruby 2.3.0](https://github.com/packsaddle/ruby-git_diff_parser/commit/89c1293c34dd3dd3947f153d5244c58c0dfb4caf)
- (d982f1e) [chore(test): update rubies](https://github.com/packsaddle/ruby-git_diff_parser/commit/d982f1e250d760021bb87e03f6363c1eb412e6e2)
- (3f71f23) [chore(test): use 1.9.3](https://github.com/packsaddle/ruby-git_diff_parser/commit/3f71f239944c7a1953593fa326372980827203b7)
- (4d99841) [Merge pull request #245 from packsaddle/chore/travis](https://github.com/packsaddle/ruby-git_diff_parser/commit/4d998416f4dc5604826b84b7663dd7fe65c19d53)
- (6c7249b) [docs(changelog): build](https://github.com/packsaddle/ruby-git_diff_parser/commit/6c7249b5f03edf2029b6bc47e0230cacb4e13cc0)
- (7367699) [chore(app): bump version to v3.2.0](https://github.com/packsaddle/ruby-git_diff_parser/commit/7367699766b2e225e9a8e4eef8b47d5a57b7e5fe)


</details>